.platform.app.yaml

```diff
@@ -147,12 +147,8 @@ web:
         '^/sites/default/files/(css|js)':
           expires: 2w
 
-# Set up migration content refresh every night with 30 minute gap between
-# file sync and db restoration on this app.
+# Set up migration content refresh every night.
 crons:
-  d7files:
-    spec: '30 0 * * *'
-    cmd: 'migrate-scripts/sync-files.sh $D7_PROJECT_ID $D7_PROJECT_ENV'
```

Comment:
DItched as we're going to download the files we need during the migration itself, from the publicly accessible paths... eg: `https://www.northernireland.gov.uk/sites/default/files/publications/some-doc.pdf`

This saves a whole lot of disk space by not needing us to copy the files from the legacy project into the file mount which migrate would then move/copy the file from into the D9 `public://` namespace/storage area.